
#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_maintainer_mode.dpatch by  <rodrigo@nul-unu.com>
##
## DP: Use AM_MAINTAINER_MODE, to avoid re-running autotools
## DP: except when explicitely called in debian/rules

@DPATCH@
diff -urNad agg-2.5+dfsg~/configure.in agg-2.5+dfsg/configure.in
--- agg-2.5+dfsg~/configure.in 2007-07-21 19:58:06.966030999 -0500
+++ agg-2.5+dfsg/configure.in  2007-07-21 19:58:33.595695250 -0500
@@ -2,7 +2,7 @@
 AC_CANONICAL_TARGET
 AC_CONFIG_HEADERS(include/config.h)
 AM_INIT_AUTOMAKE(agg, 2.5.0)
-
+AM_MAINTAINER_MODE
 
 dnl Checks for programs.
 AC_PROG_CC

